apps/trustlab/src/payload/fields/fullTitle/index.js

```diff
@@ -0,0 +1,20 @@
+import { deepmerge } from "@mui/utils";
```

Comment:
If these components/fields/blocks are the same in charterafrica, codeforafrica, etc., can't we add them to `@commons-ui/payload` package and reuse (not necessarily in this PR)?

Reply:
Agreed @kilemensi. However, there are a few differences due to the Payload CMS version. We can start with TrustLab, then update the others once we've migrated them to Payload 3

Reply:
#1150